Core/Debugger/WebSocket/HLESubscriber.cpp

```diff
@@ -247,20 +247,24 @@ void WebSocketHLEFuncAdd(DebuggerRequest &req) {
 	u32 addr;
 	if (!req.ParamU32("address", &addr))
 		return;
+
 	u32 size = -1;
 	if (!req.ParamU32("size", &size, false, DebuggerParamType::OPTIONAL))
 		return;
 	if (size == 0)
 		size = -1;
 
+	addr &= ~0x3; // Align.
+	size = (size + 3) & ~0x3; // Align.
+
 	std::string name;
 	if (!req.ParamString("name", &name, DebuggerParamType::OPTIONAL))
 		return;
 	if (name.empty())
 		name = StringFromFormat("z_un_%08x", addr);
 
 	u32 prevBegin = g_symbolMap->GetFunctionStart(addr);
-	u32 endBegin = size == -1 ? prevBegin : g_symbolMap->GetFunctionStart(addr + size - 1);
+	u32 endBegin = size == -1 ? prevBegin : g_symbolMap->GetFunctionStart(addr + size - 4);
```

Comment:
Let's say the user only submitted the correctly aligned addresses and sizes to `hle.func.add`. Why do we now pass `addr + size - 4` instead of the original `addr + size -1`. You didn't change the `SymbolMap` methods.